ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -45,14 +44,8 @@ public EthSigner(final Config config, final RunnerBuilder runnerBuilder) {
   }
 
   public void run() {
-    // set log level per CLI flags
```

Comment:
Does this get added back elsewhere? ...as I didn't spot it as an addition.

_If_ it's gone, does the log level still get set appropriately?

Reply:
Looks like an entire directory of files were lost - oops?

Reply:
😂